app/views/providers/limitations/show.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= page_template(
+      page_title: t('.h1-heading')
+    ) %>
+
+<% if @legal_aid_application.proceeding_types.present? %>
```

Comment:
If no proceeding types are present ....... when can this occur? 

If it does occur, how does the user move on from this page as the "continue" button is within the conditional block?